tests/test_server.py

```
@@ -232,6 +232,28 @@ async def test_prompts():
     assert "escalation_summary" in prompt_names
 
 
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "name,arguments",
```

Ruff (0.16.0)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)